### PR TITLE
imageset transfer learning notebook with Keras API and imageset

### PR DESCRIPTION
**Pull request by hhbyyh**
imageset transfer learning notebook with Keras API and imageset

### CODE REVIEW

File: apps/image_transfer_learning/transfer-learning_dataframe.ipynb [hunk 1]
```diff
@@ -0,0 +1,757 @@
+{
```
Comment (jason-dai) [inline apps/image_transfer_learning/transfer-learning_dataframe.ipynb]: change to transfer-learning_nnframe?

File: apps/image_transfer_learning/transfer_learning_imageSet_keras.ipynb [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.5.3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (jason-dai) [inline apps/image_transfer_learning/transfer_learning_imageSet_keras.ipynb]: add some visualizations

File: apps/image_transfer_learning/transfer-learning_dataframe.ipynb (hunk 1 above)
Comment (hkvision) [reply apps/image_transfer_learning/transfer-learning_dataframe.ipynb]: change the folder name to `image-transfer-learning` to keep the same with other apps?

Comment (hhbyyh) [reply apps/image_transfer_learning/transfer-learning_dataframe.ipynb]: Updated. Thanks.

**Comment by hhbyyh** [comment]
Thanks for the review. Added some images: https://github.com/intel-analytics/analytics-zoo/blob/d0312468370cd90da3ff461815b6f2894ee083bf/apps/image-transfer-learning/transfer_learning_imageset_keras.ipynb

**Pull request closed without merging**